packages/core/helper-plugin/src/components/EmptyBodyTable.tsx

```diff
@@ -1,11 +1,15 @@
 import React from 'react';
 
 import { Box, Flex, Loader, Tbody, Td, Tr } from '@strapi/design-system';
-import PropTypes from 'prop-types';
 
-import { EmptyStateLayout } from './EmptyStateLayout';
+import { EmptyStateLayout, EmptyStateLayoutProps } from './EmptyStateLayout';
 
-const EmptyBodyTable = ({ colSpan, isLoading, ...rest }) => {
+export interface EmptyBodyTableProps extends Omit<EmptyStateLayoutProps, 'hasRadius' | 'shadow'> {
+  colSpan: Td['colSpan'];
+  isLoading?: boolean;
+}
```

Comment:
```suggestion
export interface EmptyBodyTableProps extends Omit<EmptyStateLayoutProps, 'hasRadius' | 'shadow'>, Pick<TdProps, 'colSpan'> {
  isLoading?: boolean;
}
```

I think you might need to import the type not the component?

Reply:
easier to pick more I guess? Imo I think it's best to use the utilities we have as opposed to accessing, it also means we typically don't have to think about the requirements on the prop.